### PR TITLE
Add python implemetation

### PR DESCRIPTION
**Pull request by elchupanebrej**
### 🤔 What's changed?

Added support of Python language as part of an integration with https://github.com/elchupanebrej/pytest-bdd-ng

### 📋 Checklist:

- [x] I agree to respect and uphold the [Cucumber Community Code of Conduct](https://cucumber.io/conduct/)
- [x] I've changed the behaviour of the code
  - [x] I have added/updated tests to cover my changes.
- [x] My change requires a change to the documentation.
  - [x] I have updated the documentation accordingly.
- [x] Users should know about my change
  - [x] I have added an entry to the "Unreleased" section of the [**CHANGELOG**](../CHANGELOG.md), linking to this pull request.

----

*This text was originally generated from a [template](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/about-issue-and-pull-request-templates), then edited by hand. [You can modify the template here.](https://github.com/cucumber/.github/edit/main/.github/PULL_REQUEST_TEMPLATE.md)*

### CODE REVIEW

File: .github/workflows/release-python.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+on:  # yamllint disable-line rule:truthy
+  push:
+    branches:
+      - release/*
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
```
Comment (olleolleolle) [inline .github/workflows/release-python.yaml:13]: Could use checkout@v3

File: python/pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,66 @@
+[project]
+name = "gherkin-ci-environment"
+description = "This library detects the CI environment based on environment variables defined by CI servers."
+authors = [
+  {name = "Cucumber Limited", email = "cukes@googlegroups.com"},
+]
+maintainers = [
+  {name="Konstantin Goloveshko" , email = "kostya.goloveshko@gmail.com"},
```
Comment (olleolleolle) [inline python/pyproject.toml:8]: Should there be a space around all `=` signs?

File: python/pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+]
+license = {text = "MIT"}
+urls = {Repository ="https://github.com/cucumber/ci-environment"}
```
Comment (olleolleolle) [inline python/pyproject.toml:11]: Should there be a space around all = signs?

File: .github/workflows/release-python.yaml (hunk 1 above)
Comment (elchupanebrej) [reply .github/workflows/release-python.yaml]: Fixed

**Comment by elchupanebrej** [comment]
Linter fixups are not relevant for python implementation and are moved into separate PR: https://github.com/cucumber/ci-environment/pull/210

**Comment by elchupanebrej** [comment]
Build fixed: https://github.com/elchupanebrej/ci-environment/pull/1 
Please re-run workflows

**Comment by aslakhellesoy** [comment]
Hi @elchupanebrej,

Thanks for your making your first contribution to Cucumber, and welcome to the Cucumber committers team! You can now push directly to this repo and all other repos under the cucumber organization! 🍾

In return for this generous offer we hope you will:

* ✅ Continue to use branches and pull requests. When someone on the core team approves a pull request (yours or someone else's), you're welcome to merge it yourself.
* 💚 Commit to setting a good example by following and upholding our [code of conduct](https://cucumber.io/conduct) in your interactions with other collaborators and users.
* 💬 Join the [community Slack channel](http://cucumber.io/support#slack) to meet the rest of the team and make yourself at home.
* ℹ️ Don't feel obliged to help, just do what you can if you have the time and the energy.
* 🙋 Ask if you need anything. We're looking for feedback about how to make the project more welcoming, so please tell us!

On behalf of the Cucumber core team,
Aslak Hellesøy
Creator of Cucumber

File: .github/workflows/release-python.yaml [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        with:
+          python-version: "3.11"
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip build twine
+      - name: Build and publish
+        working-directory: ./python
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.PYPI_TOKEN }}
```
Comment (mpkorstanje) [inline .github/workflows/release-python.yaml:24]: @blaisep can you remove this file from main and resubmit it in a PR? 

We currently don't have anything to publish to PyPI. Once that is setup the PR to enable python releases can be made.

**Pull request merged**